RUNNING.md

```diff
@@ -0,0 +1,72 @@
+# Running
+
+## Table of Contents
+
+- [Running With Docker](#running-with-docker)
+- [Environment Variables](#environment-variables)
+- [Troubleshooting](#troubleshooting)
+
+## Running With Docker
+
+For convenience,  `docker-compose.yml` configuration files are supplied with this project.
+The compose file orchestrates the Publisher application and the EQ schema validation service.
+A benefit of this approach is that there is no need to run the schema validation service manually.
+
+### Environment Variables
+
+| Name                   | Description                             | Required |
+| ---------------------- | --------------------------------------- | -------- |
+| `REACT_APP_API_URL`    | Set Author API URL                      | Yes      |
+| `REACT_APP_LAUNCH_URL` | Set the launch-a-survey target          | No       |
+| `PUBLIC_URL`           | The public URL inferred if not provided | No       |
+
+From a command window(s) navigate to the component directory and run the command to start up the component.
+
+| Component Directory   | Command                           | Browser                                           |
+| -                                     | -                                           | -                                                 |
+| eq-author                      | `yarn start`                         | [http://localhost:3000](http://localhost:3000)    |
+| eq-author-api               | `docker-compose up --build`  | [http://localhost:4000/graphiql](http://localhost:4000/graphiql)                                                              |
+| eq-author-publisher     | `docker-compose up --build`  |  [http://localhost:9000](http://localhost:9000)                                   |
+|                                                                                              | [/graphql/:questionaireId](http://localhost:9000/graphql/1)   Demonstrates the JSON that is output by the Author API |
+|                                                                                              | [/publish/:questionaireId](http://localhost:9000/publish/1)    Demonstrates the published EQ JSON                     |
```

Comment:
I think this might be clearer as a step-by-step instructions rather than this tabular format.